package.json

```diff
@@ -93,7 +93,7 @@
     "source-map-support": "^0.5.12",
     "ts-node": "^8.1.0",
     "tsd": "^0.22.0",
-    "typescript": "^4.1.0"
+    "typescript": "4.7.4"
```

Comment:
Caret let 4.8.2 of typescript into the mix. This version has breaking changes, which were causing ts compiler complaints on npm install of `@slack/deno-slack-sdk`